app/views/data_lists/_form.html.erb

```diff
@@ -60,14 +60,16 @@
             <div class="field">
               <%= f2.label :measurements %><br>
               <%= f2.collection_select :measurement_ids, Measurement.where(universe: 'UHERO').all.order(:prefix, :data_portal_name), :id, :prefix_and_name, {:include_blank => false, selected: nil} %>
-            </div>
-            <div class="actions">
-              <%= f2.submit 'Add Measurement' %>
+              <span class="actions"><%= f2.submit 'Add Measurement' %></span>
+              <br/>or <%= link_to 'Create a new UHERO measurement for this data list',
+                             {controller: :measurements, action: :new, universe: 'UHERO', data_list_id: @data_list.id} %>*
+              <br/>or <%= link_to 'Create a new DBEDTCOH measurement for this data list',
+                             {controller: :measurements, action: :new, universe: 'DBEDTCOH', data_list_id: @data_list.id} %>*
             </div>
         <% end %>
         <br />
   <% end %>
-
+<p>* If you click a link before saving this screen, entered data may be lost.</p>
```

Comment:
I think this note is a good sign we shouldn't have the measurement creation links in this view.

Reply:
I think copying the new measurements/edit layout, would make it easier to see tell the user which parts may lose data. (since all the measurement actions are automatically saved).